contributing.rst

```diff
@@ -1576,15 +1576,46 @@ Video files should be put in the :file:`/vid/` directory in the source, and they
 
 The length of the videos must be less than a minute.
 
-There is no ``video`` directive to add a video, so to add a video in a document, you can do the following:
+We have a custom ``video`` directive to add a video.
```

Comment:
There are custom rst directives and roles for writing about rst roles and directives.

http://www.sphinx-doc.org/en/stable/domains.html#the-restructuredtext-domain

(I've used them other places in the contrib guide, so you can see how they normally appear.)

Reply:
```

.. rst:directive:: .. video:: path/to/video
 
  Explanation

  ..rst:role:: autoplay

    Explanation of autoplay


```